### DISCUSSION TITLE
PS1 Adapter

### DISCUSSION THREAD

**Discussion by haydnkilyan** [darthcloud/BlueRetro#661] [Q&A]
So I see that a PS1 external adapter can mimic a multi tap.... my question is, do I need to by 2 adapters to play 2 player games or can I just use 1 in 'multi tap' mode?

**Comment by mknrls** [comment]
I don’t see why this wouldn’t work but definitely need to able it in the config.

I do also wonder if you put a physical controller in Port2 if it would re-assign the port numbers..  @darthcloud could you chimed in ?

**Comment by darthcloud** [comment]
A few games like Crash Team Racing support multitap on port one. So for those few games if you enable BlueRetro multitap mode then you can play multiplayer with a single dongle.

But for most games you will need either 2 dongle or a BlueRetro with 2 controller plug.

**Comment by haydnkilyan** [comment]
Thank you both for answering
